### PR TITLE
Filter preview results per field

### PR DESCRIPTION
**Pull request by gotcha**

### CODE REVIEW

File: collective/searchandreplace/browser/searchreplacetable.pt [hunk 1]
```diff
@@ -1,3 +1,57 @@
...
+      tal:attributes="value field/value;
+                      checked python:field.value in filter_fields"
+    />
+    <span tal:replace="field/label" />
+  </span>
+  </tal:fields>
+  </div>
+
+  <input type="submit" id="form.actions.Preview" name="form.actions.Preview"
+  value="Aperçu" class="button">
```
Comment (spereverde) [inline collective/searchandreplace/browser/searchreplacetable.pt:51]: default should be english,  i18n:attributes needed

**Review by spereverde — CHANGES_REQUESTED** [review]
- small i18n fix needed
- general question: when you click the filter, the results list does not auto-update, you have to manually re-submit the form, probably more difficult to make this work, but in our use case the data is pretty big, so redoing a search takes a while for the user
- everything else seems to be working fine :-)

**Comment by gotcha** [comment]
>     * small i18n fix needed
> 
>     * general question: when you click the filter, the results list does not auto-update, you have to manually re-submit the form, probably more difficult to make this work, but in our use case the data is pretty big, so redoing a search takes a while for the user
> 
>     * everything else seems to be working fine :-)

- i18n fix done.
- the results do not auto-update because filtering is costly as it has to be done server side. UI would not be very responsive if a new search was launched for each click.
- filtering needs to happen server side because of batching, the results tied to the fields that the user selects might not exist in the current batch (hope this makes sense).

**Review by mauritsvanrees — APPROVED** [review]
I tried it out and looks good to me.

**Review by spereverde — APPROVED** [review]
LGTM, i18n requested changes are done

File: collective/searchandreplace/browser/searchreplacetable.pt (hunk 1 above)
Comment (spereverde) [reply collective/searchandreplace/browser/searchreplacetable.pt]: i18n changes are done

**Comment by gotcha** [comment]
Rebased and merged locally

**Pull request closed without merging**